appendix-basic65.tex

```diff
@@ -187,30 +192,57 @@ \subsection{BASIC 65 Constants}
 \subsection{BASIC 65 Variables}
 \index{BASIC 65 Variables}
 
-Each scalar variable consumes 8 bytes of storage in memory.
-The reserved area in bank 0 from \$F700-\$FEFF can store 256 variables.
-Variables don't need to be declared, and their type is determined by an appended
-character. All variables without an appended character are regarded as REAL by default,
-and storage is claimed at their first usage. They are also initialised to zero,
-whereas string variables are initialised as an empty string "".
-
-All 104 one-letter variables are declared as {\bf fast} variables.
-26 user functions are declared as {\bf fast} functions.
```

Comment:
Ok, see in your new text you've made mention of fast variables, but not fast functions.

ASIDE: perhaps tidbits like this could be placed in some 'BASIC optimisation tips' section one day.

E.g. another potential BASIC optimisation I recall was a tip that `fredrikr` gave me: to speed up your BASIC programs, any frequently called subroutines should be place either:
- after (but near to) their usage/call points
- if they must be earlier than the callers, place them at the start of your program (as he mentioned basic would search for the line number by looking at the first line of the program and onwards)

Reply:
Added a note that the fast-variable idea also applies to function names.